web/controllers/homepage_controller.ex

```diff
@@ -56,9 +57,9 @@ defmodule App.HomepageController do
 
     content_filter = Enum.filter_map(content, true_tuples, second_value)
 
-    article_query = create_tag_query(tag, "articles")
+    article_query = Resources.create_tag_query(tag, "articles")
 
-    link_query = create_tag_query(tag, "resources")
+    link_query = Resources.create_tag_query(tag, "resources")
```

Comment:
why don't we pass down `"resource"` instead of `"resources"` (similar for `"articles"`), it's easier to handle adding an `"s"` to the end than having a `String.slice(type, 0..-2)` just to remove the trailing `"s"`